src/main/java/domain/LadderResult.java

```diff
@@ -0,0 +1,18 @@
+package domain;
+
+import java.util.List;
+
+public class LadderResult {
+
+    private final List<String> gameResults;
+
+    public LadderResult(List<String> gameResults) {
+        this.gameResults = gameResults;
+    }
+
+    public List<String> getResults(){
+        return gameResults;
+    }
+
+
+}
```

Comment:
```suggestion
    public List<String> getResults() {
        return gameResults;
    }
}
```
사소하지만 컨벤션 항상 지켜주세요.
intellij에서 제공하는 자동 정렬 기능도 꼭 활용해보세요!

Reply:
https://dandev.tistory.com/entry/%EA%B8%B0%ED%83%80-%EC%9D%B8%ED%85%94%EB%A6%AC%EC%A0%9C%EC%9D%B4IntelliJ-%EC%BD%94%EB%93%9C-%EC%A0%95%EB%A0%AC-%EB%8B%A8%EC%B6%95%ED%82%A4Windows